### PR TITLE
Spelling fixes

### PR DESCRIPTION
**Pull request by nschonni**
Ran cspell to fix up some typos. Not entirely done, but I'll try and add it to the build with the config separately

### CODE REVIEW

File: examples/menubar/menubar-1/mb-outreach.html [hunk 1]
```diff
@@ -18,7 +18,7 @@ <h1 id="outreach">Outreach</h1>
         <h2 id="comm-engage">Community Engagement</h2>
         <p><a href="menubar-1.html#code-ex-1">Back to <code>menubar</code> example</a></p>
 
-        <h2 id="copr-rel">Corporate Relations</h2>
+        <h2 id="corp-rel">Corporate Relations</h2>
```
Comment (jnurthen) [inline examples/menubar/menubar-1/mb-outreach.html]: I'm not sure idrefs should be spellchecked.

Comment (nschonni) [reply examples/menubar/menubar-1/mb-outreach.html]: They can be indvidually ignored, but in this case there are no refs https://github.com/w3c/aria-practices/search?q=copr&unscoped_q=copr and it looked like a typo

Comment (mcking65) [reply examples/menubar/menubar-1/mb-outreach.html]: looks like we have some unnecessary files left over from old content in the menubar-1 folder.

**Comment by mcking65** [comment]
@nschonni, thank you for all these fixes!

Having integrated spell checking would be great. I don't know anything about cspell. Having to ignore ids individually sounds like serious pain ... makes me wonder if cspell is a good solution for our purposes. Seems like we would not want to spell check the values of any attributes except for perhaps aria-label. The validator will catch most any other attribute mistake.

File: examples/menubar/menubar-1/mb-outreach.html (hunk 1 above)
Comment (jnurthen) [reply examples/menubar/menubar-1/mb-outreach.html]: Agree - looks like we can remove mb-research.html, mb-arts.html and mb-outreach.html

Comment (mcking65) [reply examples/menubar/menubar-1/mb-outreach.html]: Done in commit c57878b

**Pull request merged**